ci/lint/06_script.sh

```diff
@@ -7,7 +7,9 @@
 export LC_ALL=C
 
 if [ "$TRAVIS_EVENT_TYPE" = "pull_request" ]; then
-  test/lint/commit-script-check.sh $TRAVIS_COMMIT_RANGE
+  MERGE_BASE=$(git merge-base HEAD master)
```

Comment:
I think you want to use $TRAVIS_BRANCH here instead of master, as that would be wrong for non-master PRs.

Also, I think this can be simplified to just `COMMIT_RANGE="$TRAVIS_BRANCH..HEAD"` then.

Reply:
done, I have also introduced $TRAVIS_BRANCH to `lint-git-commit-check.sh` where I had previously removed TRAVIS_COMMIT_RANGE